content/en/blog/2022-02-09-security-the-value-of-sboms/index.md

```diff
@@ -0,0 +1,166 @@
+> specific part, the BOM makes it easy to locate affected products.
+
+For the Flux project we publish a Software Bill of Materials (SBOM) with
+each release. The SBOM is generated with
+[Syft](https://github.com/anchore/syft) in the
+[SPDX](https://spdx.dev/) format.
+
+![SPDX logo](featured-image.png)
+
+The `spdx.json` file is available for download on the GitHub release page
+e.g.:
+
+```shell
+curl -sL https://github.com/fluxcd/flux2/releases/download/v0.25.3/flux\_0.25.3\_sbom.spdx.json | jq
+```
+
+Inspecting the JSON data, you will see that for each of the files and
+libraries required for building and shipping the release you can verify
+the license, origin, version and checksum.
+
+What might seem like a lot of overhead and unnecessary bookkeeping,
+quickly turns out as useful information because it allows you to
+
+- Verify the origin and integrity of artifacts
+- Inspect the dependencies easily for CVEs and known security issues
+- Get a holistic view of your complete supply chain, so which other
+  dependencies and Open Source projects now become part of your
+  stack
+
+Because it is structured data, all of the above can be done in an
+automated, programmatic fashion.
+
+Big organizations, corporate or governmental, already keep track of
+SBOMs and make decisions based on the information provided there. Some
+started requiring SBOMs for software in-use.
+
+### Possible use-cases for SBOMs
+
+Here are a couple more concrete examples of what the SBOMs for Flux
+allow you to do:
+
+- Security alerts of dependencies will be the most obvious use-case.
+  If a CVE is detected, you can inspect your SBOM and see if the
+  components you are using are in any way affected.
+- It will be easy to identify when a certain component was created and
+  how. In addition to that, having the licensing information of all
+  the pieces, you better know what you can do with it (redistribute,
+  change, etc).
+- Automation allows you to send alerts, if a compliance issue is
+  detected, e.g. if the licensing of updated/replaced dependencies
+  changes.
+- Missing components or required build files. Incompatible licenses
+  and more.
+
+One example of automating all of this could be to store SBOMs in <https://grafeas.io/>. This way you could search across your Estate for:
+
+- Find all images that are built from a particular Github commit that is known to have introduced a security problem.
+- Find all images that were built by a certain version of a certain builder when that builder is known to have been compromised.
+- Find all images in my project that are impacted by `CVE-1234`.
+
+For policy enforcement, combining SBOMs with grafeas and [kritis](https://github.com/grafeas/kritis) would enable auto-blocking of any application that is vulnerable to a specific CVE for example. This is the basis of <https://cloud.google.com/binary-authorization/>, which goes a step further from SBOM and also touches attestation.
```

Comment:
```suggestion
For policy enforcement, [kritis](https://github.com/grafeas/kritis) can be used to leverage the information provided by SBOMs inside Grafeas to enforce policies inside of a cluster, enabling auto-blocking of applications that are vulnerable to a specific CVE for example.
```

I am not sure we should mention attestation at this point in time. Potentially later on referring to some things we implement in the Flux project.